api/types/types.go

```diff
@@ -76,14 +84,17 @@ type Image struct {
 	Labels      map[string]string
 }
 
+// GraphDriverData returns Image's graph driver config info
+// when calling inspect command
 type GraphDriverData struct {
 	Name string
 	Data map[string]string
 }
 
+// ImageInspect contains response of Remote API:
 // GET "/images/{name:.*}/json"
 type ImageInspect struct {
-	Id              string
+	ID              string
```

Comment:
Same as above, as it is *serialized* as JSON, I think it would be better to do something like :

```go
ID string `json:Id`
```